modules/gapi/test/cpu/gapi_core_tests_fluid.cpp

```diff
@@ -119,6 +148,24 @@ INSTANTIATE_TEST_CASE_P(AddWeightedTestFluid, AddWeightedTest,
                                 Values(CORE_FLUID),
                                 Values(Tolerance_FloatRel_IntAbs(1e-5, 2).to_compare_obj())));
 
+INSTANTIATE_TEST_CASE_P(NormTestFluid, NormTest,
+                        Combine(Values(CV_8UC1, CV_16UC1, CV_16SC1),
+                                Values(cv::Size(1280, 720),
+                                       cv::Size(640, 480),
+                                       cv::Size(128, 128)),
+                                Values(-1),
+                                Values(CORE_FLUID),
+                                Values(AbsToleranceScalar(1e-5).to_compare_obj()),
+                                Values(NORM_INF, NORM_L1, NORM_L2)));
+
+INSTANTIATE_TEST_CASE_P(IntegralTestFluid, IntegralTest,
+                        Combine(Values(CV_8UC1, CV_16UC1, CV_16SC1),
+                                Values(cv::Size(1280, 720),
+                                       cv::Size(640, 480),
+                                       cv::Size(128, 128)),
+                                Values(-1),
+                                Values(CORE_FLUID)));
```

Comment:
We don't have Integral kernel in Fluid.

And when this kernel is not found, G-API takes the OpenCV version by default.

We need to forbid G-API taking the default kernel package when running accuracy test - otherwise, we can't guarantee that a proper kernel is used.

Please turn all `CORE_FLUID`, `IMGPROC_FLUID`, and other macros for all backend's accuracy tests to use `cv:;gapi::use_only` [modifier](https://docs.opencv.org/4.2.0/d6/d25/structcv_1_1gapi_1_1use__only.html)

Reply:
Please also note - this `use_only` should apply to accuracy tests only! Other functional and unit tests can still rely on the default kernel package.

Reply:
Fixed. 
Added DISABLED_ for tests with unsupported kernels.

Reply:
Cool! Thanks